mappings/net/minecraft/entity/Entity.mapping

```diff
@@ -287,6 +343,7 @@ CLASS aif net/minecraft/entity/Entity
 	METHOD bP getEntityWorld ()Lbhh;
 	METHOD bQ getServer ()Lnet/minecraft/server/MinecraftServer;
 	METHOD bR isImmuneToExplosion ()Z
+	METHOD bS entityDataRequiresOperator ()Z
```

Comment:
`grantsOperatorCommandAccess`? A similar method is present in `BlockEntity` as well. How is it named?

Reply:
`BlockEntity` doesn't seem to have such a method.
Also, `grantsOperatorCommandAccess` sounds like, when using this item, you get operator command access.

Reply:
This is from CommandBlockItem, without looking at much else.
```java
@Nullable
@Override
protected BlockState getBlockState(ItemPlacementContext itemPlacementContext) {
	PlayerEntity playerEntity3 = itemPlacementContext.getPlayer();
	return (playerEntity3 == null || playerEntity3.isCreativeLevelTwoOp()) ? super.getBlockState(itemPlacementContext) : null;
}
    }
```